integration-tests/src/nft_marketplace.rs

```diff
@@ -0,0 +1,136 @@
+#![cfg(test)]
```

Comment:
if we have pallet specific tests we could separate them one level deeper into a folder